CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v0.4.1
```

Comment:
Our convention is to put new changes under ## vNext until we cut a release.

Reply:
Will update